src/Ormolu/Exception.hs

```diff
@@ -84,7 +86,7 @@ withPrettyOrmoluExceptions m = m `catch` h
           OrmoluParsingFailed _ _ -> 3
           OrmoluOutputParsingFailed _ _ -> 4
           OrmoluASTDiffers _ _ -> 5
-          OrmoluNonIdempotentOutput _ _ _ -> 6
```

Comment:
Can you change the other branches as well?

Reply:
Done